python/cudnn/gemm/frost/kernel_templates/sm100_block_scale_matmul_2ctamma_static.py

```
@@ -285,9 +301,12 @@ def _kernel(
 
     M = m
     N = n
-    num_tile_m = cute.ceil_div(M, cgrp_tile_mnk[0])
-    num_tile_n = cute.ceil_div(N, cgrp_tile_mnk[1])
     num_k_tiles = cute.ceil_div(k, cgrp_tile_mnk[2])
```

Ruff (0.16.1)

[error] Undefined name `cgrp_tile_mnk`

(F821)